bentoml/archive/loader.py

```diff
@@ -21,7 +21,7 @@
 import tempfile
 import logging
 
-from bentoml.utils.usage_stats import track_loading
+from bentoml.utils.usage_stats import track_loading, track_load_start
```

Comment:
suggest rename: track_loading => track_load_finish, no need to change the actual tag in amptitude